src/main/javacc/copy.jjt

```diff
@@ -41,39 +40,40 @@ SKIP :
 
 TOKEN : /* KEYWORDS */
 {
-  <K_STDIN: "STDIN">
```

Comment:
did you look into using the [IGNORE_CASE] qualifier here?

Reply:
Done.